### PR TITLE
Navigation(desktop) grid

### PR DESCRIPTION
**Pull request by prankurpandeyy**

### CODE REVIEW

**Comment by Naman-Saxena1** [comment]
Your nav bar should have 100% width and in your grid 70-30 and 30-70 example, image is still of same size, you will need to look into it.

File: components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="grid.css">
+</head>
+<body>
+    <div class="high-container">
+
+
+   
+    <h3>50:50 Grid example</h3>
+    <div class="grid-50-50">
+        <img src="/images/scorpio-sign.jpg" alt="">
```
Comment (Shivani-1524) [inline components/Grid/grid.html:17]: add accessibility text in your alt attribute

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]
please recheck your deployed netlify preview link

**Comment by nikhiljugale007** [comment]
Image in grid is not taking full width of grid.
Overall UI can be improved.

File: components/Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <img src="/images/scorpio-sign.jpg" alt="">
```
Comment (gauravsinhaweb) [inline components/Grid/grid.html:18]: give proper naming in alt

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/Grid/grid.html (hunk 1 above)
Comment (prankurpandeyy) [reply components/Grid/grid.html:17]: sure ,will do that now .

File: components/Grid/grid.html (hunk 2 above)
Comment (prankurpandeyy) [reply components/Grid/grid.html:18]: sure ,will do that now .

**Pull request merged**